packages/dev-utils/src/generate-documentation-table.ts

```diff
@@ -412,18 +412,10 @@ type IncludeEntry = {
       "A key which will embedded in all the bundled files. The SDK will be able to use the key to apply filtering rules, for example using the `thirdPartyErrorFilterIntegration`.",
   },
   {
-    name: "_experiments",
-    type: "string",
+    name: "injectBuildInformation",
+    type: "boolean",
     fullDescription:
-      "Options that are considered experimental and subject to change. This option does not follow semantic versioning and may change in any release.",
-    children: [
-      {
-        name: "injectBuildInformation",
-        type: "boolean",
-        fullDescription:
-          "If set to true, the plugin will inject an additional `SENTRY_BUILD_INFO` variable. This contains information about the build, e.g. dependencies, node version and other useful data.\n\nDefaults to `false`.",
-      },
-    ],
+      "If set to true, the plugin will inject an additional `SENTRY_BUILD_INFO` variable. This contains information about the build, e.g. dependencies, node version and other useful data.\n\nDefaults to `false`.",
```

Comment:
I was just able to visualize this a bit better now that you changed it and realized that we should probably still leave `injectBuildInformation` in experimental state. Sorry for all the back and fourth!

Reply:
No problem, just a quick revert!